### ISSUE TITLE
Text colors in div.mw-message-box in dark theme fail to meet WCAG AA level

### ISSUE THREAD

**Issue by Dringsim** [StarCitizenTools/mediawiki-skins-Citizen#639]
In page“test”,
```
<div class="mw-message-box-success mw-message-box"><p>success [[test]] [[首页]] [[www]]</p></div>
<div class="mw-message-box-warning mw-message-box"><p>warning [[test]] [[首页]] [[www]]</p></div>
<div class="mw-message-box-error mw-message-box"><p>error [[test]] [[首页]] [[www]]</p></div>
<div class="mw-message-box"><p>info [[test]] [[首页]] [[www]]</p></div>
```
results (page “首页” exists, while “www” not):
![image](https://github.com/StarCitizenTools/mediawiki-skins-Citizen/assets/57343841/07ae065f-a1a5-4f87-99e7-e08fd55a348a)
Contrast ratio:
|class|background color|--color-base--emphasized|--color-link|--color-link-new|
|---|---|---|---|---|
|mw-message-box-success|--background-color-success|3.90|1.24|1.27|
|mw-message-box-warning|--background-color-warning|3.91|1.24|1.27|
|mw-message-box-error|--background-color-destructive|5.69|1.81|1.85|
|(mw-message-box)|--color-surface-2|13.62|4.33|4.43|

**Comment by alistair3149** [comment]
Thanks for the report! I'm aware is the issue and opting to revamp the state background colors soon. Would be looking into it at the end of the week
